pipelines/measurement/measurement_tract_table.yaml

```diff
@@ -33,4 +33,38 @@ tasks:
         # config.referenceCatalogLoader.colorterms.load(os.path.join(getPackageDir('obs_subaru'),
         #                                                            'config',
         #                                                            'colorterms.py'))
-        # config.instrument: 'hsc'
\ No newline at end of file
+        # config.instrument: 'hsc'
+  TE1:
+    class: lsst.faro.measurement.TractTableMeasurementTask
+    config:
+      connections.package: validate_drp
+      connections.metric: TE1_table
+      python: |
+        from lsst.faro.measurement import TExTableTask
+        config.measure.retarget(TExTableTask)
+        # config.measure.raColumn = "ra"
+        # config.measure.decColumn = "decl"
+        # config.columns = ["coord_ra", "coord_dec", "deblend_nChild"]
+        # config.columnsBand = ["ixx", "iyy", "ixy", "ixxPSF",
+        #                       "iyyPSF", "ixyPSF", "extendedness", "psfFlux",
+        #                       "psfFluxErr"]
+        config.measure.minSep = 0.25
+        config.measure.maxSep = 1.0
+        config.measure.shearConvention = False
+  TE2:
+    class: lsst.faro.measurement.TractTableMeasurementTask
+    config:
+      connections.package: validate_drp
+      connections.metric: TE2_table
+      python: |
+        from lsst.faro.measurement import TExTableTask
+        config.measure.retarget(TExTableTask)
+        # config.measure.raColumn = "ra"
```

Comment:
Why is this commented out portion here, and should the config.measure.decColumn be "decl"? (it might be but I have only seen "coord_dec")

Reply:
Also left-over from debugging, have been removed.